build_projects/Microsoft.DotNet.Cli.Build.Framework/Enumerations/BuildArchitecture.cs

```diff
@@ -3,6 +3,8 @@ namespace Microsoft.DotNet.Cli.Build.Framework
     public enum BuildArchitecture
     {
         x86 = 1,
-        x64 = 2
+        x64 = 2,
+        arm = 3,
+        arm64 = 4
```

Comment:
I cound not add the function because I was not sure what you were using. Do you use `TARGETPLATFORM` environment for recognizing arm64? If you share your build command for ARM64, I can add the `Isarm64()` function.